packages/desktop-client/src/components/mobile/accounts/Accounts.tsx

```diff
@@ -307,6 +351,27 @@ function AllAccountList({
             getAccountBalance={getAccountBalance}
             onOpenAccount={onOpenAccount}
           />
+          {closedAccounts.length > 0 && (
+            <AccountHeader
+              id="closed"
+              name={showClosedAccounts ? t('Closed') : t('Closed...')}
+              pressHandler={onToggleClosedAccounts}
+              amount={getClosedAccountsBalance()}
+              style={{ marginTop: 30 }}
+              hideChevron={true}
+            />
+          )}
+          {showClosedAccounts && (
+            <AccountList
+              aria-label={t('Closed accounts')}
+              accounts={closedAccounts}
+              getAccountBalance={getAccountBalance}
+              onOpenAccount={onOpenAccount}
+              ref={el => {
+                if (el) closedAccountsRef.current = el;
+              }}
```

Comment:
```suggestion
              ref={closedAccountsRef}
```

Reply:
Keeping this as-is, because it doesn't work correctly otherwise.